telegram/_chatadministratorrights.py

```diff
@@ -148,6 +178,9 @@ def __init__(
         self.can_post_messages: Optional[bool] = can_post_messages
         self.can_edit_messages: Optional[bool] = can_edit_messages
         self.can_pin_messages: Optional[bool] = can_pin_messages
+        self.can_post_stories: Optional[bool] = can_post_stories
+        self.can_edit_stories: Optional[bool] = can_edit_stories
+        self.can_delete_stories: Optional[bool] = can_delete_stories
         self.can_manage_topics: Optional[bool] = can_manage_topics
 
         self._id_attrs = (
```

Comment:
I guess the id attrs (and the corresponding docstring) should be updated as well. This is in line with our stability policy 😎

Reply:
can you remind me why we are adding optional attributes to _id_attrs again?

Reply:
Mh, I had indeed missed that only some if the attributes are optional. But then again, we already started with them, so we might as well continue...